pygetty/formatters.py

```diff
@@ -7,6 +7,17 @@
 MASTERY_DIMENSIONS_REGEX = re.compile(r'(?P<width>[0-9]+)x(?P<height>[0-9]+)')
 
 
+def format_image(image):
+    if 'date_created' in image:
+        image['date_created'] = pendulum.parse(image['date_created'])
+
+    if 'keywords' in image:
```

Comment:
@klardotsh do we use width/height anywhere?  Would it make sense to add as parameters?

Reply:
That stuff exists in `image['max_dimesions']['width' or 'height']`, already provided by Getty.